### PR TITLE
Update to add Inovio's new vaccine trial to treatments.md

### PR DESCRIPTION
**Pull request by esell17**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by cbrueffer** [comment]
Very nice read, thanks @esell17! Could you please convert this to one sentence per line as per https://github.com/greenelab/covid19-review/blob/master/USAGE.md#manubot-markdown? This will make future editing and reviewing easier.

**Comment by esell17** [comment]
Thank you, @cbrueffer! Just converted them to one sentence per line. Newbie here :) I'll do this going forward

File: content/20.treatments.md [hunk 1]
```diff
@@ -495,19 +495,32 @@ Vaccines, widely recognized as one of the most significant advances in human hea
...
 ##### Anticipated Mechanism
 
-Why it may be useful
+This approach may offer several advantages over traditional vaccination approaches, such as the stimulation of both B- as well as T-cell responses and the absence of any infectious agent. 
 
 ##### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway [@https://clinicaltrials.gov/ct2/show/NCT04336410]. 
+The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers.
```
Comment (rando2) [inline content/20.treatments.md:507]:
```suggestion
The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers. 
Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner. 
The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response. 
It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers [@doi:10.1016/j.coi.2011.03.008].
```
Suggestion to move things around

File: content/20.treatments.md [hunk 2]
```diff
@@ -495,19 +495,32 @@ Vaccines, widely recognized as one of the most significant advances in human hea
...
-Why it may be useful
+This approach may offer several advantages over traditional vaccination approaches, such as the stimulation of both B- as well as T-cell responses and the absence of any infectious agent. 
 
 ##### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway [@https://clinicaltrials.gov/ct2/show/NCT04336410]. 
+The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers. 
+The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method, [@doi:10.4161/hv.24702].
+
```
Comment (rando2) [inline content/20.treatments.md:509]:
```suggestion
```

File: content/20.treatments.md [hunk 3]
```diff
@@ -495,19 +495,32 @@ Vaccines, widely recognized as one of the most significant advances in human hea
...
+Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner.
```
Comment (rando2) [inline content/20.treatments.md:510]:
```suggestion
```

File: content/20.treatments.md [hunk 4]
```diff
@@ -495,19 +495,32 @@ Vaccines, widely recognized as one of the most significant advances in human hea
...
 
 ##### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway [@https://clinicaltrials.gov/ct2/show/NCT04336410]. 
+The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers. 
+The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method, [@doi:10.4161/hv.24702].
+
+Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner. 
+The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response.
```
Comment (rando2) [inline content/20.treatments.md:511]:
```suggestion
```

File: content/20.treatments.md [hunk 5]
```diff
@@ -495,19 +495,32 @@ Vaccines, widely recognized as one of the most significant advances in human hea
...
+Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway [@https://clinicaltrials.gov/ct2/show/NCT04336410]. 
+The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers. 
+The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method, [@doi:10.4161/hv.24702].
+
+Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner. 
+The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response. 
+It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers [@doi:10.1016/j.coi.2011.03.008].
```
Comment (rando2) [inline content/20.treatments.md:512]:
```suggestion
```

**Comment by rando2** [comment]
@esell17 make sure you add yourself to the [author list](https://github.com/greenelab/covid19-review/blob/master/content/metadata.yaml)

**Pull request merged**